src/main/scala/sigmastate/types.scala

```diff
@@ -349,7 +349,14 @@ case object SBigInt extends SPrimType with SEmbeddable with SNumericType {
   }
 }
 
-case object SGroupElement extends SProduct with SPrimType with SEmbeddable {
+case object SString extends SProduct {
+  override def ancestors: Seq[SType] = Nil
+  override def methods: Seq[SMethod] = Seq()
+  override type WrappedType = String
+  override val typeCode: TypeCode = 101: Byte
+}
```

Comment:
@ergomorphic to choose a type code I just searched for max one already in use and incremented it. Does it sound good? It's quite an error prone technique. Should we at least gather them at one place (like `OpCodes`)?